pages/game.tsx

```diff
@@ -1,24 +1,67 @@
-import React from "react";
-import Link from "next/link";
-import { Container } from "react-bootstrap";
+import Board from "../components/Board";
+import Keyboard from "../components/Keyboard";
+import { createContext, useState } from "react";
+import { boardDefault } from "../components/words";
 
-export default function game() {
-    return (
-        <Container>
-            <h1>Hello, world!</h1>
-            <Link href="/about">About</Link>
-            <h1 class="text-centered">
-                ⬜⬜⬜⬜⬜ <br></br>
-                ⬜⬜⬜⬜⬜ <br></br>
-                ⬜⬜⬜⬜⬜ <br></br>
-                ⬜⬜⬜⬜⬜ <br></br>
-                ⬜⬜⬜⬜⬜ <br></br>
-            </h1>
-            <h2 class="text-centered">
-                Q W E R T Y U I O P <br></br>
-                A S D F G H J K L <br></br>
-                Z X C V B N M <br></br>
-            </h2>
-        </Container>
-    );
+export const GameContext = createContext(null);
+
+function game() {
+  const [board, setBoard] = useState(boardDefault);
+  const [currAttempt, setCurrAttempt] = useState({
+    rowIndex: 0,
+    letterIndex: 0,
+  });
+  const onEnter = () => {
+    if (currAttempt.letterIndex != 5) return;
+    setCurrAttempt({
+      rowIndex: currAttempt.rowIndex + 1,
+      letterIndex: 0,
+    });
```

Comment:
**readiness for change/fitness for purpose**: how are you guys going to end up handling the case where all six attempts have been exhausted (i.e. `rowIndex == 6` or smth)? I feel like onEnter should do something different here? Or maybe you just disable it altogether?